source/includes/_aviate-catalog.md

```diff
@@ -145,8 +146,210 @@ Represents information on whether a tenant uses an XML or Aviate catalog. It has
+    "availableForBps": [],
+    "availableAddons": [],
+    "retiredDate": ""
+  },
+  {
+    "name": "standard",
+    "prettyName": "",
+    "category": "BASE",
+    "availableForBps": [],
+    "availableAddons": [],
+    "retiredDate": ""
+  }
+]
+```
+
+**Query Parameters**
+
+| Name                                      | Type   | Required | Default | Description                                                                          |
+|-------------------------------------------|--------|----------| ---- |--------------------------------------------------------------------------------------
+| **accountId**                             | UUID   | false    | none | Account Id. If specified, returns plans for the specified `tenantId` and `accountId` |
+
+
+**Response**
+
+If successful, returns a status code of 200 and a `ProductData` list.
 
-## Catalog APIs
+
+### Get Plans Per Product
+
+Returns the plans corresponding to a product. Includes retired plans.
+
+**HTTP Request**
+
+`GET /plugins/aviate-plugin/v1/catalog/{productName}/plansPerProduct`
+
+> Example Request:
+
+```shell
+curl -X GET \
+-H"Authorization: Bearer ${ID_TOKEN}" \
+-H 'Content-Type: application/json' \
+-H 'X-killbill-apiKey: alphaF' \
+-H 'X-killbill-apisecret: alphaF' \
+'http://127.0.0.1:8080/plugins/aviate-plugin/v1/catalog/standard/plansPerProduct' 
+```
+
+```java
+```
+
+```ruby
+```
+
+```python
+```
+
+````php
+````
+
+````javacript
+````
+
+> Example Response:
+
+```json
+[
+  {
+    "name": "standard-monthly",
+    "prettyName": "standard-monthly",
+    "recurringBillingMode": "IN_ADVANCE",
+    "effectiveDate": "2025-04-15T06:46:02.000Z",
+    "effectiveDateForExistingSubscriptions": "",
+    "productName": "standard",
+    "pricelistName": "DEFAULT",
+    "retired": false,
+    "phases": [
+      {
+        "prettyName": "",
+        "type": "EVERGREEN",
+        "durationUnit": "UNLIMITED",
+        "durationLength": -1,
+        "fixedPrices": [],
+        "recurringPrices": {
+          "billingPeriod": "DAILY",
+          "prices": [
+            {
+              "currency": "USD",
+              "value": "30.000000000"
+            }
+          ]
+        },
+        "usages": []
+      }
+    ]
+  }
+]
+```
+
+**Query Parameters**
+
+| Name                                      | Type   | Required | Default | Description                                                                          |
+|-------------------------------------------|--------|----------| ---- |--------------------------------------------------------------------------------------
+| **accountId**                             | UUID   | false    | none | Account Id. If specified, returns plans for the specified `tenantId` and `accountId` |
+
+
+**Response**
+
+If successful, returns a status code of 200 and a `PlanData` list.
+
+### Delete Product
+
+Deletes the specified product. If `deleteAllPlans` is specified, deletes all the plans corresponding to the product.
```

Comment:
Does this `retire` the associated plans? Or fully `delete`? Would be helpful to clarify this for the user.